browser/ephemeral_storage/BUILD.gn

```diff
@@ -10,4 +10,8 @@ source_set("ephemeral_storage") {
     "//content/public/browser",
     "//net",
   ]
+
+  if (is_android) {
+    deps += [ "//chrome/browser/ui" ]
```

Comment:
I'm confused, this is needed with and without android

Reply:
Okay. I've fixed this.

Reply:
where have you fixed this? I don't see any changes

Reply:
I have fixed it locally, but I will push a new version of the branch once I've responded to all of @iefremov's comments.